sdk/js/src/ReactNativeStore.ts

```diff
@@ -0,0 +1,58 @@
+import { BucketedUserConfig, DVCLogger } from '@devcycle/types'
+import { DVCPopulatedUser } from './User'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { StoreKey } from './Store'
+
+export class ReactNativeStore {
+    store: typeof AsyncStorage
+    logger?: DVCLogger
+
+    constructor(logger: DVCLogger) {
+        this.store = AsyncStorage
+        this.logger = logger
+    }
+
+    save(storeKey: string, data: unknown): void {
+        this.store.setItem(storeKey, JSON.stringify(data))
+    }
+
+    load(storeKey: string): any {
+        let storedValue
+        (async () => {
```

Comment:
what's the reasoning behind this? is this going to make it an async function, why can't it just be an async function?